app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationTableFooter.jsx

```diff
@@ -43,18 +44,24 @@ export default function RegistrationAdministrationTableFooter({
           registrations.length - newcomerCount
         } Returners = ${registrations.length} People`}
       </Table.Cell>
-      <Table.Cell>{`${countryCount}  Countries`}</Table.Cell>
-      <Table.Cell />
-      { competitionInfo['using_payment_integrations?'] && <Table.Cell>{moneyCountHumanReadable(registrations, competitionInfo)}</Table.Cell>}
-      { eventsToggled ? competitionInfo.event_ids.map((evt) => (
+      <Table.Cell>{`${countryCount} Countries`}</Table.Cell>
+      <Table.Cell key="registered on" />
```

Comment:
Out of interest: Why are you passing in a key outside of a loop?

Reply:
Basically as an equivalent to
```
// this footer cell corresponds to the 'registered on' column
<Table.Cell />
```
since comments have a tendency to be ignored/missed and get out-of-date. Obviously this could still happen, but it seems less likely.